index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+    hello world
+    hello world
+
+    -->
+    <h1 id="title">Internet speed</h1>
+
+    <p id="description">
+      We are providing faster speed for everyone across the world.
+    </p>
+
+    <form id="survey-form">
+      <label id="name-label">Your name</label>
+      <input type="text" id="name" placeholder="name" required /><br />
+      <label id="email-label">Your E-mail</label>
+      <input type="email" id="email" placeholder="email" required /><br />
+      <label id="number-label">Your number</label>
+      <input
+        type="number"
+        id="number"
+        min="1"
+        max="11"
+        placeholder="number"
+      /><br />
+      <label id="speed">speeds</label>
+      <select id="dropdown">
+        <option value="10mbps">10 mbps</option>
+        <option value="11mbps">11 mbps</option>
+        <option value="12mbps">12 mbps</option>
+        <option value="13mbps">13 mbps</option></select
+      ><br />
+      <label id="gender-label-male">male</label>
+      <input type="radio" name="gender" id="male" value="male" />
+      <label id="gender-label-female">female</label>
+      <input type="radio" name="gender" id="female" value="female" /><br />
+
+      <input type="checkbox" name="visa" id="visa" value="visa" /> visa
+      <input type="checkbox" name="paypal" id="paypal" value="paypal" />paypal
+      <input
+        type="checkbox"
+        name="mastercard"
+        id="mastercard"
+        value="mastercard"
+      />mastercard <br />
+      <label id="coments">coments
```

Comment:
please close the <label> tag.unclosing the tag makes your code not function well.